bundles/org.openhab.binding.qbus/README.md

```diff
@@ -1,10 +1,10 @@
 # Qbus Binding
 
-![Qbus Logo](doc/Logo.JPG)
+![Qbus Logo](doc/qbus.JPG)
```

Comment:
You can remove the logo here entirely as you uploaded it to the docs repo. The website will put the logo here automatically.

Reply:
ok, done. Took a look at other bindings and noticed there was no doc folder. So i removed mine as well. Hope this is ok?